api/src/Bookmark/Model/Bookmark.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spira\Bookmark\Model;
```

Comment:
Yea I see the thinking, but revisionable is actually an extension to a dependency, I feel like these features are just standard models, otherwise we should be moving the more generic core models like `User` to this directory.